### PR TITLE
Style submit button in change password

### PR DESCRIPTION
**Pull request by yeegor**

### CODE REVIEW

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4724192&bid=15772622)

File: src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.MyAccountPasswordForm-Button {
+    @include mobile {
+        width: 100%;
+    }
+    position: relative;
+    left: 50%;
+    transform: translateX(-50%);
+}
```
Comment (Danpiel) [inline src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4724192)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4724192&bid=15772622)

File: src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss (hunk 1 above)
Comment (Danpiel) [inline src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4724192)

File: src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.MyAccountPasswordForm-Button {
+    @include mobile {
+        width: 100%;
+    }
+    position: relative;
```
Comment (alfredsgenkins) [inline src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss]: Separate all your style blocks with new lines. Make sure there is an additional new-line in the end.

File: src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.MyAccountPasswordForm-Button {
```
Comment (alfredsgenkins) [inline src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss]: Prefer declaration of block separately

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724192&bid=15772622)

File: src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
+.MyAccountPasswordForm {
+    
+    &-Button{
```
Comment (Danpiel) [inline src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss]: Codacy Issue found: [Unexpected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724192)

File: src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.MyAccountPasswordForm {
+
```
Comment (Danpiel) [inline src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724192)

File: src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss (hunk 4 above)
Comment (Danpiel) [inline src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724192)

**Comment by negzu** [comment]
Changes were applied :)

**Comment by yeegor** [comment]
In ticket it is shown that button is not styled properly not bcs it has no center alignment but bcs it has no class responsible for color, size etc. Should I center all the buttons in popups or should I leave everything as it is now and reverse centering this button?

**Comment by alfredsgenkins** [comment]
Wrong button styled.

**Pull request closed without merging**